lib/ansible/modules/network/checkpoint/cp_mgmt_access_rule.py

```diff
@@ -0,0 +1,332 @@
+    type: bool
+  time:
+    description:
+      - List of time objects. For example, "Weekend", "Off-Work", "Every-Day".
+    type: list
+  track:
+    description:
+      - Track Settings.
+    type: list
+    suboptions:
+      accounting:
+        description:
+          - Turns accounting for track on and off.
+        type: bool
+      alert:
+        description:
+          - Type of alert for the track.
+        type: str
+        choices: ['none', 'alert', 'snmp', 'mail', 'user alert 1', 'user alert 2', 'user alert 3']
+      enable_firewall_session:
+        description:
+          - Determine whether to generate session log to firewall only connections.
+        type: bool
+      per_connection:
+        description:
+          - Determines whether to perform the log per connection.
+        type: bool
+      per_session:
+        description:
+          - Determines whether to perform the log per session.
+        type: bool
+      type:
+        description:
+          - a "Log", "Extended Log", "Detailed  Log", "None".
+        type: str
+  user_check:
+    description:
+      - User check settings.
+    type: list
+    suboptions:
+      confirm:
+        description:
+          - N/A
+        type: str
+        choices: ['per rule', 'per category', 'per application/site', 'per data type']
+      custom_frequency:
+        description:
+          - N/A
+        type: list
+        suboptions:
+          every:
+            description:
+              - N/A
+            type: int
+          unit:
+            description:
+              - N/A
+            type: str
+            choices: ['hours', 'days', 'weeks', 'months']
+      frequency:
+        description:
+          - N/A
+        type: str
+        choices: ['once a day', 'once a week', 'once a month', 'custom frequency...']
+      interaction:
+        description:
+          - N/A
+        type: str
+  vpn:
+    description:
+      - Communities or Directional.
+    type: list
+  comments:
+    description:
+      - Comments string.
+    type: str
+  details_level:
+    description:
+      - The level of detail for some of the fields in the response can vary from showing only the UID value of the object to a fully detailed
+        representation of the object.
+    type: str
+    choices: ['uid', 'standard', 'full']
+  ignore_warnings:
+    description:
+      - Apply changes ignoring warnings.
+    type: bool
+  ignore_errors:
+    description:
+      - Apply changes ignoring errors. You won't be able to publish such a changes. If ignore-warnings flag was omitted - warnings will also be ignored.
+    type: bool
+extends_documentation_fragment: checkpoint_objects
+"""
+
+EXAMPLES = """
+- name: add-access-rule
+  cp_mgmt_access_rule:
+    action: Apply Layer
+    inline_layer: Inline
+    layer: Network
+    name: Rule 1
+    position: 1
+    state: present
+
+- name: set-access-rule
+  cp_mgmt_access_rule:
+    action: Ask
+    action_settings:
```

Comment:
@chkp-orso I can see u have defined `action_settings` as list and here in example u have written it as Dict, plz change it to list, otherwise this play Ansible will throw error:
```suggestion
    action_settings:
    - enable_identity_captive_portal: true
      limit: Upload_1Gbps
```